packages/react-storage/src/components/StorageBrowser/Views/Controls/Controls.tsx

```diff
@@ -15,6 +15,7 @@ import { SummaryControl } from './Summary';
 import { TableControl } from './Table';
 import { TargetControl } from './Target';
 import { TitleControl } from './Title';
+import { TableV2 } from './TableV2';
```

Comment:
Let's not add this to the `Controls` interface quite yet

Reply:
got it, just removed